scripts/populate_sm_analysis.py

```diff
@@ -0,0 +1,70 @@
+"""
+Script to just populate analysis entries with status=completed in the 
+sample-metadata DB.
```

Comment:
Could you add more description on why this script might be used? Is it a one off script to populate old data, or run at the end of a job?

Reply:
Yeah, it's one-off. I'll add a comment